src/css/style.scss

```diff
@@ -164,9 +178,17 @@
     height: 24px;
     margin-left: 23px;
 
-    @media screen and (min-width: $break-small) {
+    @media screen and (min-width: $break-tablet) {
       display: none;
     }
+
+    &--white {
```

Comment:
Please, give a more abstract name instead of `white`. Maybe `light`?

Reply:
Changed.